### ISSUE TITLE
code-oss repo is mostly empty

### ISSUE THREAD

**Issue by ghost** [headmelted/code-builds#10]
Need at least a cursory link back to code.headmelted.com and maybe the
[article outlining the motivations].

I'd also expect that the repo contains the build infrastructure laid bare.

[article outlining the motivations]:
https://headmelted.com/why-we-need-the-best-tools-on-chromebooks-and-raspberry-pi-3ab4c10a611c

**Comment by headmelted** [comment]
Hi, this is something I absolutely intend to do - at this point in time it still requires (minimal) manual intervention from time-to-time which I'm working to remove from the process.

There are two obstacles at the moment.

The least of these is an utter lack of time on my part lately (couldn't be avoided, but now improving so shouldn't be a problem going forward).

The biggest problem is that I still intend to merge my build script additions upstream into Microsoft's github codebase - I initially spoke to them about this prior to starting work on this last year. The issue is thst I can't do this for the same reason that the code isn't on this github presently, which is that Travis doesn't have the embedded Debian package repositories I'm using on their whitelist. I raised this with them late last year, but they (politely it must be said) let me know that they have a freeze on white-listing at the moment which meant I had to implement a custom solution.  Once this changes, I'll rework the scripts into Travis YAML that can sit side-by-side with Microsoft's builds, and this project will no longer be necessary i.e. ARM support will be in the core! :-).
